resources/instructions/createSObjectLwcQuickActions.html

```diff
@@ -0,0 +1,81 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Offline Starter Kit: Create SObject LWC Quick Actions</title>
+    </head>
+    <style>
+        h1 {
+            font-size: 16px;
+        }
+        h2 {
+            font-size: 14px;
+        }
+        table {
+            border-collapse: collapse;
+        }
+        table,
+        th,
+        td {
+            border-width: 1px;
+            border-style: solid;
+        }
+        th,
+        td {
+            text-align: center;
+        }
+    </style>
+    <body>
+        <h1>Create SObject LWC Quick Actions</h1>
+
+        <p>
+            The following sObjects are present in your configured landing page:
```

Comment:
nit: Probably standardize on SObjects? I'm not sure which capitalization is actually preferred but SObject seems to be used throughout.

Reply:
I had done it as a distinction between title case (headings) and lower case (in sentences). But I think for title case, I erroneously looked at [the `SObject` class](https://developer.salesforce.com/docs/atlas.en-us.apexref.meta/apexref/apex_methods_system_sobject.htm) in the Apex Reference Guide, which is likely taking casing queues based more on coding standards than nomenclature.

The [Use sObjects](https://trailhead.salesforce.com/content/learn/modules/apex_database/apex_database_sobjects) trail in Trailhead seems to treat it as "sObject" in all cases. So that's probably the one worth standardizing on. Good callout!